evm/src/cpu/kernel/tests/bn254.rs

```diff
@@ -0,0 +1,188 @@
+use std::str::FromStr;
+
+use anyhow::Result;
+use ethereum_types::U256;
+
+use crate::bn254::{
+    curve_generator, fp12_to_vec, frob_fp12, gen_fp12, gen_fp12_sparse, miller_loop, mul_fp12,
+    power, tate, twisted_curve_generator, Curve, Fp12, TwistedCurve,
+};
+use crate::cpu::kernel::aggregator::KERNEL;
+use crate::cpu::kernel::interpreter::run_interpreter;
+
+fn make_label(lbl: &str) -> U256 {
+    U256::from(KERNEL.global_labels[lbl])
+}
+
+fn make_stack(vecs: Vec<Vec<U256>>) -> Vec<U256> {
+    let mut stack = vec![];
+    for vec in vecs {
+        stack.extend(vec)
+    }
+    stack
+}
```

Comment:
Honestly, I'd just call `.extend` a few times instead of using this function. It'd be more immediately readable.